src/cr/cube/matrix/assembler.py

```diff
@@ -168,6 +168,34 @@ def means(self):
         """
         return self._assemble_matrix(self._measures.means.blocks)
 
+    @lazyproperty
+    def pairwise_significance_p_vals(self):
+        """3D optional np.float64 ndarray of overlaps-p_vals matrices for each subvar.
+
+        Raises `ValueError if the cube-result does not include `overlaps`
+        and `valid_overlaps` cube-measures.
+        """
+        return np.array(
+            [
+                self._assemble_matrix(self._measures.get_p_vals_for(subvar_idx).blocks)
+                for subvar_idx in range(len(self.column_labels))
+            ]
+        )
+
+    @lazyproperty
+    def pairwise_significance_t_stats(self):
+        """3D optional np.float64 ndarray of overlaps-t_stats matrices for each subvar.
+
+        Raises `ValueError if the cube-result does not include `overlaps`
+        and `valid_overlaps` cube-measures.
+        """
+        return np.array(
+            [
+                self._assemble_matrix(self._measures.get_t_stats_for(subvar_idx).blocks)
+                for subvar_idx in range(len(self.column_labels))
+            ]
+        )
+
```

Comment:
Can you explain why we need 2 different property for tstats and pvals? They should come from the `PairwiseTtest` object isn't it. This way we compute the calc 2 times just for accessing different values. Is there a reason for doing this in separate props?

Reply:
Yes. The reason is to not force the client to inspect the objects we return. It makes much more sense from the client perspective to use something like `pairwise_t_stats` than it does to use `pairwise_sig_tests.t_stats`. We might discuss how to optimize, but from API POW, it clearly warrants two different properties, afaics.

Reply:
The old way (where we wrap them together) is more convenient for us, but it's ugly for the clients (e.g. exporter).

Reply:
also resolving this for now, might get to the "optimize" part later, I still have a bunch of steve's comments to address